src/python_file_finder.rs

```diff
@@ -61,9 +61,8 @@ fn build_walker(
 
     walk_builder
         .types(build_types(ignore_notebooks).unwrap())
+        .standard_filters(use_git_ignore)
```

Comment:
https://docs.rs/ignore/0.4.23/ignore/struct.WalkBuilder.html#method.standard_filters

All the options that `standard_filters` set are enabled by default anyway, so this does not change the current behaviour when not overriding exclusion list. But when overriding it, this allows disabling everything related to ignore (because there actually is more than `.gitignore` at play).